src/ngraph_mark_for_clustering.cc

```diff
@@ -570,6 +582,15 @@ Status MarkForClustering(Graph* graph) {
     bool mark_for_clustering = false;
 
     do {
+      // check if output node
+      bool skip_it = false;
+      TF_RETURN_IF_ERROR(CheckIfOutputNode(node, skip_these_nodes, skip_it));
+      if (skip_it) {
+        NGRAPH_VLOG(5) << "Found Output Node: " << node->name()
```

Comment:
should we reduce the log level here?

Reply:
I did not want a lot of logs to be displayed at the lower log level.